README.md

```diff
@@ -9,3 +9,9 @@
 
 ## Run Samples Based On Spring Native in Spring Boot 2
 Two Maven profiles have been defined in this project to support compiling Spring applications to native executables: `buildpack` and `native`. The `buildpack` profile will use [Buildpacks](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-buildpacks) and the `native` profile will use [Native Build Tools](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-native-build-tools). Please follow the [storage-blob-native sample](spring-native/storage-blob-native) for more details.
+
+
+## How to run samples with Spring Boot 3?
```

Comment:
How about deleting `HOW_TO_RUN_SAMPLES_WITH_SPRING_BOOT_3.md` and paste the contents here?

```
## Samples for Spring Boot 3
1. In each sample's README.md, there is content like this:
> Current sample is using Spring Cloud Azure 4.x (which is compatible with Spring Boot 2.x). If you want the sample about Spring Cloud Azure 6.x (which is compatible with Spring Boot 3.x), please refer to [CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md](./CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md)
```
If you are adding a new sample, here is the template file about CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md: [./CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md](CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md)